From 56f60d0ddf99bd2c9791eeb12139d8ece535a3b5 Mon Sep 17 00:00:00 2001
From: =?utf-8?q?Rafa=C3=ABl=20Carr=C3=A9?= <rafael.carre@gmail.com>
Date: Mon, 7 Dec 2009 04:42:06 +0100
Subject: [PATCH 5/7] UNIX: search for plugins in $PREFIX/lib/convert-freebox

---
 configure.ac |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/configure.ac b/configure.ac
index a2c8b74..e24affa 100644
--- a/configure.ac
+++ b/configure.ac
@@ -7,7 +7,7 @@ VERSION_MAJOR="1"
 VERSION_MINOR="1"
 VERSION_REVISION="0"
 VERSION_EXTRA="-git"
-PKGDIR="vlc"
+PKGDIR="convert-freebox"
 AC_SUBST(PKGDIR)
 
 CONFIGURE_LINE="`echo "$0 $ac_configure_args" | sed -e 's/\\\/\\\\\\\/g'`"
-- 
1.6.3.3

